IdentityCore/src/webview/embeddedWebview/challangeHandlers/MSIDPKeyAuthHandler.m

```diff
@@ -72,6 +73,22 @@ + (BOOL)handleChallenge:(NSString *)challengeUrl
     NSMutableURLRequest *responseReq = [[NSMutableURLRequest alloc] initWithURL:responseUrlComp.URL];
     [responseReq setValue:kMSIDPKeyAuthHeaderVersion forHTTPHeaderField:kMSIDPKeyAuthHeader];
     [responseReq setValue:authHeader forHTTPHeaderField:MSID_OAUTH2_AUTHORIZATION];
+
+    BOOL refreshTokenIsValid = NO;
+    if (customHeaders != nil && customHeaders.count > 0)
+    {
+        NSString *refreshToken = [customHeaders objectForKey:MSID_REFRESH_TOKEN_CREDENTIAL];
```

Comment:
nit: Based on the discussion in scrum, you may want to add comment or variable name specifying that this is a PRT header for readability